axon-framework/events/event-processors/README.md

```diff
@@ -307,12 +307,12 @@ A `JpaSequencedDeadLetterQueue` configuration example:
 ```java
 public class DeadLetterQueueExampleConfig {
     
-    public ConfigurerModule configureDeadLetterQueueFor(String processingGroup) {
+    public ConfigurerModule configureDeadLetterQueueFor(EventProcessingConfigurer configurer) {
```

Comment:
I think the non-Spring configuration wouldn't have to construct a `ConfigurerModule` at all.
That's a specific solution for within Spring environments to add configuration methods on the fly.

In the Java Config world, I'd expect a `void` method that receives the `EventProcessingConfigurer` upon which it invokes the configuration changes.